public/index.html

```diff
@@ -5,17 +5,38 @@
         <title>Chat</title>
         <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.1/angular.min.js"></script>
         <script src="js/main.js"></script>
+        <script src="js/chatService.js"></script>
```

Comment:
Sorry, should have said this yesterday. You'll want to look into a bundler. I think the others are using webpack. That will mean you don't have to reference all of your scripts individually.